kafka/src/test/java/org/axonframework/extensions/kafka/eventhandling/util/KafkaContainerClusterTest.java

```diff
@@ -0,0 +1,18 @@
+package org.axonframework.extensions.kafka.eventhandling.util;
+
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+/**
+ * A cluster of kafkas using test container.
```

Comment:
I'd more so anticipate something like this:

```suggestion
 * A {@link KafkaContainerCluster} set up by using {@link Testcontainers.
```

And similarly, some author tags. Grab those creds :p